### PR TITLE
Better POD for usage

### PR DESCRIPTION
**Pull request by magicDGS**
A bit of refactoring and fixing of the POD for the printing the usage.

### CODE REVIEW

**Comment by magicDGS** [comment]
@robmaz - maybe you can have a look to this idea: it will help to have a better help message. Just run `./bin/distmap --help` to check.

**Comment by magicDGS** [comment]
This is prepared for review, @robmaz!

File: bin/distmap [hunk 1]
```diff
@@ -455,272 +455,351 @@ sub check_inputs {
...
 
 
 =head1 AUTHORS
 
- Ram Vinay Pandey
- Christian Schloetterer
+The original code for distmap was written by Ram Vinay Pandey and
+Christian Schloetterer as described in the citation.
+
+Version 3 is a major improvement of the code developped by Rupert
```
Comment (robmaz) [inline bin/distmap]: one "p" too many

Comment (magicDGS) [reply bin/distmap]: Did you fixed this? Sorry that I didn't catch it before merging...

Comment (magicDGS) [reply bin/distmap]: PR for the fix -> #74 (merged directly)

**Pull request merged**